src/pages/Profile/Profile.vue

```diff
@@ -101,12 +101,40 @@
           </div>
           <!-- second widget -->
           <h1>Key Performance Area</h1>
-          <AreaComponent/>
```

Comment:
what use of areacompoent and where it is now?
send all kpi/ era data to this component via prop